index.bs

```diff
@@ -315,6 +315,30 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
+<h3 id="promises">Use Promises</h3>
```

Comment:
I'm almost tempted to make it "Design asynchronous APIs" instead 😄.  I guess I'll stick with "Designing", though.